content/applications/finance/accounting/taxes.rst

```diff
@@ -307,6 +307,31 @@ directly to a product line.
    .. image:: taxes/list-sequence.png
       :alt: The taxes' sequence in Odoo determines which tax is applied first
 
+Extra taxes
+===========
+
+"Extra taxes" is a broad term referring to additional taxes beyond the standard or basic taxes
+imposed by governments. These extra taxes can be **luxury** taxes, **environmental** taxes,
```

Comment:
Below, you speak about ecotax; is it another name for the environmental tax? Should we always use the same name to refer to this tax?